src/teleop/jetson/jetson_teleop.py

```diff
@@ -189,18 +191,20 @@ def ra_open_loop_control(self, msg: Joy) -> None:
         left_trigger = raw_left_trigger if raw_left_trigger > 0 else 0
         raw_right_trigger = msg.axes[self.xbox_mappings["right_trigger"]]
         right_trigger = raw_right_trigger if raw_right_trigger > 0 else 0
-
         self.ra_cmd.velocity = [
-            self.ra_config["joint_a"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x"),
-            self.ra_config["joint_b"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_y"),
-            self.ra_config["joint_c"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_y"),
-            self.ra_config["joint_d"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_x"),
-            self.ra_config["joint_e"]["multiplier"] * (right_trigger - left_trigger),
-            self.ra_config["joint_f"]["multiplier"]
+            self.ra_config["joint_a"]["slow_mode_multiplier" if self.ra_slow_mode else "multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x"),
```

Comment:
Hmm I don't think I was clear when explaining. We should always want to multiply by the multiplier. I think we should have for loop that iterates over every name in RA_NAMES and
1. calculates the normal multiply times the joystick axis
2. if in slow mode, multiplies by the slow mode multiplier